UI/window-basic-preview.cpp

```diff
@@ -1861,6 +1861,18 @@ static void DrawStripedLine(float x1, float y1, float x2, float y2,
 		float dx;
 		float dy;
 
+#if defined(_M_ARM64)
+		if (x1 < x2) {
+			dx = (std::min)(xx1 + 7.5f * offX, x2);
+		} else {
+			dx = (std::max)(xx1 + 7.5f * offX, x2);
+		}
+		if (y1 < y2) {
+			dy = (std::min)(yy1 + 7.5f * offY, y2);
+		} else {
+			dy = (std::max)(yy1 + 7.5f * offY, y2);
+		}
+#else
```

Comment:
See previous comment about changes related to `std::max`/`std::min`

Reply:
I've provided a brief explanation about the `std::max`/`std::min` modifications in the previous comment. You can review it [here](https://github.com/obsproject/obs-studio/pull/10997#discussion_r1688385006)